controller/backup_volume_controller.go

```diff
@@ -320,7 +320,7 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 		backupLabelMap := map[string]string{}
 
 		backupURL := backupstore.EncodeBackupURL(backupName, backupVolumeName, backupTargetClient.URL)
-		if backupInfo, err := backupTargetClient.BackupGet(backupURL, backupTargetClient.Credential); err != nil {
+		if backupInfo, err := backupTargetClient.BackupGet(backupURL, backupTargetClient.Credential); err != nil && !types.ErrorIsNotFound(err) {
```

Comment:
Does the controller still need to create a backup CR below if the backup info is not found?

Reply:
This change is due to I changed the behavior of `backupTargetClient.BackupGet`
it originally return nil backupInfo and nil error when it is not found
but I changed to return `notFoundError`  when it is not found so I have to skip the error to remain the same behavior.

As you said, we actually create the Backup even the backup info is not found (no `backup.cfg`) in the backupstore all the time before.
That is, in this logic, we create the Backup because we list the name (list the dir) but don't care if it has config or not.

I think we can create it, and since it won't get the backup info to sync, the status will be empty and there will be error logs in the longhorn-manager  so we and users can detect the situation.